### PR TITLE
Fixes time units for rpc client latency

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: core/src/main/java/com/google/instrumentation/stats/RpcConstants.java [hunk 1]
```diff
@@ -53,11 +53,10 @@
           "/rpc/client/response_bytes",
           "Response MB",
           MeasurementUnit.create(6, bytes));
-  // TODO(dpo): verify this encoding
   public static final MeasurementDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY =
       MeasurementDescriptor.create(
           "/rpc/client/roundtrip_latency",
-          "RPC roundtrip latency us",
+          "RPC roundtrip latency msec",
           MeasurementUnit.create(-6, seconds));
```
Comment (a-veitch) [inline core/src/main/java/com/google/instrumentation/stats/RpcConstants.java]: For millisec, shouldn't this be -3?

**Comment by dinooliva** [comment]
Good catch - thanks!

**Comment by adriancole** [comment]
Aww was hoping that was intentionally us!

**Comment by dinooliva** [comment]
Lol - gRpc may add support for finer-grain at some point but these views reflect what they have now.

**Pull request merged**